README.md

```diff
@@ -0,0 +1,17 @@
+# Integrating Kobiton with Travis CI in React Native app development pipeline
```

Comment:
Why do we even need 2 MD files? I think merging them into 1 README.md is much more simple.

Reply:
I think differently, merging into 1 file can cause some issues:
   - The readme file may very long when we keep adding content into it
   - The aim of readme file is for introducing the documentation and each md file will present the subject (help to control the reading flow)

Reply:
I think we need README.md to draw a general picture for the users. If they're interested, they will click to the doc for details information